policy_set_integration_test.go

```diff
@@ -154,6 +154,34 @@ func TestPolicySetsCreate(t *testing.T) {
 		assert.Equal(t, ps.Workspaces[0].ID, wTest.ID)
 	})
 
+	t.Run("with policies and projects provided", func(t *testing.T) {
+		skipUnlessBeta(t)
+		pTest, pTestCleanup := createPolicy(t, client, orgTest)
+		defer pTestCleanup()
+		wTest, wTestCleanup := createWorkspace(t, client, orgTest)
+		defer wTestCleanup()
+		prjTest, prjTestCleanup := createProject(t, client, orgTest)
+		defer prjTestCleanup()
+
+		options := PolicySetCreateOptions{
+			Name:       String("populated-policy-set"),
```

Comment:
Doesn't affect the test but maybe we should rename this to something else since we are already using this name for the test above.

Reply:
Done!